src/h5web/dimension-mapper/SlicingSlider.tsx

```diff
@@ -25,29 +33,34 @@ function SlicingSlider(props: Props) {
     >
       <span className={styles.label}>D{dimension}</span>
       <ReactSlider
-        // Force refresh when slider height changes - i.e. when Y-axis mapper appears/disappears
-        // https://github.com/zillow/react-slider/issues/172
-        key={`${mapperState.includes('y')}`}
+        ref={sliderRef}
         className={styles.slider}
         ariaLabel="Dimension slider"
-        trackClassName={styles.track}
-        thumbClassName={styles.thumb}
```

Comment:
I moved those class names inside the `renderThumb/Tracks` props and just reorder the other props.